packages/load-script/test/index.js

```diff
@@ -64,7 +66,7 @@ describe( 'loadScript', () => {
 			loadjQueryDependentScript( url, callback );
 
 			expect( createScriptElement ).toHaveBeenCalledTimes( 1 );
-			expect( createScriptElement ).toHaveBeenLastCalledWith( url );
+			expect( createScriptElement ).toHaveBeenLastCalledWith( url, undefined );
```

Comment:
Should we also add a test that covers when `args` is specified for `loadjQueryDependentScript`?